### PR TITLE
Updated readme to include webhook filtering

### PR DESCRIPTION
**Pull request by jamiemtdwyer**
Added a section describing how to filter webhook data when configuring webhooks.

@kevinhughes27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,9 +127,9 @@ $ rails generate shopify_app:install --api_key <your_api_key> --secret <your_app
 ```
 
 Other options include:
-* `application_name` - the name of your app, it can be supplied with or without double-quotes if a whitespace is present. (e.g. `--application_name Example App` or `--application_name "Example App"`)  
-* `scope` - the Oauth access scope required for your app, eg **read_products, write_orders**. *Multiple options* need to be delimited by a comma-space, and can be supplied with or without double-quotes  
-(e.g. `--scope read_products, write_orders, write_products` or `--scope "read_products, write_orders, write_products"`)
```
Comment (ashmaroli) [inline README.md]: The whitespaces at the end of ln#131 and #132 were deliberate. They were included to *force* the `(e.g. ` and `For more..` onto the following line when the markdown document is rendered. 
I'm okay with the deletion on ln#130.

File: README.md [hunk 2]
```diff
@@ -236,6 +236,16 @@ When the oauth callback is completed successfully ShopifyApp will queue a backgr
 
 ShopifyApp also provides a WebhooksController that receives webhooks and queues a job based on the webhook url. For example if you register the webhook from above then all you need to do is create a job called `CartsUpdateJob`. The job will be queued with 2 params `shop_domain` and `webhook` which is the webhook body.
 
+If you are only interested in particular fields, you can optionally filter the data sent by Shopify by specifying the `fields` parameter in `config/webhooks`. Note that you will still receive a webhook request from Shopify every time the resource is updated, but only the specified fields will be sent.
+
```
Comment (kevinhughes27) [inline README.md]: can you confirm if we filter the webhook send or not? I think we might.

Comment (jamiemtdwyer) [inline README.md]: Yup, I tested this and the behaviour is as written

**Comment by kevinhughes27** [comment]
:shipit:

**Comment by jamiemtdwyer** [comment]
I'll fix the whitespace and then :shipit:. Need to figure out how to prevent Atom from removing trailing whitespace...

**Pull request merged**